association-frontend/components/social-card/social-card.tsx

```diff
@@ -0,0 +1,7 @@
+type EventCardProps = {
+  component: React.ReactElement;
+};
+const SocialCard = ({ component }: EventCardProps) => {
+  return <div className="p-4 soverflow-hidden">{component}</div>;
+};
+export default SocialCard;
```

Comment:
the convention we are trying to follow in our codebase is to not use default exports, but only named exports

```suggestion
export const SocialCard = ({ component }: EventCardProps) => {
  return <div className="p-4 soverflow-hidden">{component}</div>;
};
```

(you should use default exports only in nextjs pages/XX as it's a required by nextjs)

Reply:
but if I dont put the default it will complain like: 
```
Attempted import error: '../event-card/event-card' does not contain a default export (imported as 'EventCard').
```